docs/walkthrough/integrate-with-jina.md

```diff
@@ -59,6 +59,25 @@ Text of the returned document: some text to encode
 Shape of the embedding: (768,)
 ```
 
+### Encoding a List
+Data that is stored in a regular list can be embedded in the same way you would a [DocumentArray](https://docarray.jina.ai/). Since the modality of your input data can be inferred from the model being used, there is not need to provide any additional information besides the content you want to encode:
```

Comment:
```suggestion
Data that is stored in a regular list can be embedded in the same way you would a [DocumentArray](https://docarray.jina.ai/). Since the modality of your input data can be inferred from the model being used, there is no need to provide any additional information besides the content you want to encode:
```
same here